travis-ci/cleanup-integration-tests.sh

```diff
@@ -4,31 +4,37 @@ set -e
 
 . ./travis-ci/set-env-vars.sh
 
-cd example || exit 1
-
-# Delete the stack if it's a nightly build
-if [ "$DEPLOYMENT" = "cumulus-nightly" ]; then
-  yarn
-  echo Delete app deployment
-
-  ./node_modules/.bin/kes cf delete \
-    --kes-folder app \
-    --region us-east-1 \
-    --deployment "$DEPLOYMENT" \
-    --yes
-
-  echo Delete iam deployment
-
-  ./node_modules/.bin/kes cf delete \
-    --kes-folder iam \
-    --region us-east-1 \
-    --deployment "$DEPLOYMENT" \
-    --yes
-
-  echo Delete app deployment
-else
-  yarn add @cumulus/common
-fi
-
-echo Unlocking stack
-node ./scripts/lock-stack.js false $DEPLOYMENT
\ No newline at end of file
+(
+  set -e
```

Comment:
Does this take care of ensuring the right exit code?

Reply:
If one of the commands in the subshell fails, it will stop immediately instead of trying to run the other commands, and return an error status.